docs/installation/frameworks/vuejs-v2.md

```diff
@@ -238,13 +238,57 @@ Finally, import the build in your application:
 
 Integrating the rich text editor from source allows you to use the full power of {@link framework/index CKEditor 5 Framework}.
 
+### Vite
+
+This guide assumes that you use `create-vue` as your boilerplate. To get started with Vite and Vue, run the command below.
+
+```bash
+npm init vue@latest ckeditor5-vue-example
+```
+
+This command will install and execute `create-vue`, the official project scaffolding tool for Vue. It will also allow you to customize your project, for example, by adding Typescript. Choose your preferred options.
```

Comment:
`create-vue` is creating Vite + Vue apps that are using Vue 3. This guide is about Vue 2 component. To use Vue 2.x with Vite one have to configure it manually and use https://github.com/vitejs/vite-plugin-vue2 plugin.